components/imageproc/src/meta.rs

```diff
@@ -21,7 +23,11 @@ impl ImageMeta {
     pub fn read(path: &Path) -> ImageResult<Self> {
         let reader = ImageReader::open(path).and_then(ImageReader::with_guessed_format)?;
         let format = reader.format();
-        let size = reader.into_dimensions()?;
+        let mut size = reader.into_dimensions()?;
```

Comment:
can we somehow get the exif metadata from the image object? So we don't have to read the file twice

Reply:
Yup! image-rs has a function on `ImageDecoder` to get the raw Exif data. I made `get_orientation` take the raw Exif data and updated all the upstream usages (ImageMeta, ImageOp, tests) to get it this way.